doc/explanation/microcloud.rst

```diff
@@ -26,8 +26,30 @@ Networking
 
 By default, MicroCloud uses MicroOVN for networking, which is a minimal wrapper around OVN (Open Virtual Network).
 
-MicroOVN requires an uplink network that is an actual L2 subnet (which is usually not the case in a virtual cloud environment).
-In addition, MicroOVN requires its own dedicated network interface, for example, a dedicated physical network interface, a VLAN, or a virtual function on an :abbr:`SR-IOV (Single root I/O virtualisation)`-capable network interface.
+For outside connectivity, MicroOVN requires an uplink network.
+This uplink network must be an actual L2 subnet (which is usually not the case in a virtual cloud environment).
+
+Each machine in the MicroCloud cluster must have two available network interfaces:
+
+.. _microcloud-networking-intracluster:
+
+Network interface for intra-cluster traffic
+  MicroCloud requires one network interface that is configured with an IP address of a real layer 2 network.
+  All cluster members must be connected to the same network.
```

Comment:
Is that correct?

```suggestion
  MicroCloud requires one network interface that is configured with an IP address.
  All cluster members must be connected to the same layer 2 network.
```

Reply:
How about this:

> MicroCloud requires one network interface be pre-configured with an IP within the same subnet as the other cluster members and that the network supports multicast. This is so that cluster member discovery can take place.